ea_server/api/routes.py

```diff
@@ -0,0 +1,10 @@
+import functools
+from flask import Blueprint, jsonify
+from ea_server.engine import main
+
+api_blueprint = Blueprint('api', __name__, url_prefix='/api')
```

Comment:
Maybe we should use /api/v1 prefix, so if we will change the API in the future we will have to choose between versions (i.e https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design/#:~:text=your%20caching%20system.-,Versioning%20our%20APIs,-We%20should%20have)

Reply:
You are absolutely right

Reply:
I agree too